ddp-automation/src/test/java/org/broadinstitute/ddp/tests/DatabaseUtility.java

```diff
@@ -105,8 +105,8 @@ private static void listResultSet(List<Long> resultSet) {
     public static List<Long> getAnswerIds() {
         List<Long> ids = TransactionWrapper.withTxn(handle -> {
             logger.info("Using user id: {}", getUserId());
-            JdbiAnswer answerDao = handle.attach(JdbiAnswer.class);
-            return answerDao.getAnswerIds(getUserId());
+            AnswerSql answerSql = handle.attach(AnswerSql.class);
+            return List.copyOf(answerSql.findAllAnswerIdsByOperatorUserId(getUserId()));
```

Comment:
@Rinchan20 this should provide the same functionality as what we used here before.